x-pack/packetbeat/cmd/agent.go

```diff
@@ -0,0 +1,24 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I'd prefer this code were in root.go to improve source locality. If in the future the amount of agent-specific code grows to overwhelm that file, it can be divided then.